### DISCUSSION TITLE
Pydeck event handling

### DISCUSSION THREAD

**Discussion by kerim371** [Kitware/trame#636] [Q&A] [answered]
Hi,

I'm testing the example with [Uber Pydeck](https://github.com/Kitware/trame/blob/master/examples/08_pydeck/01_uber-nyc-pickups.py).

I'm wondering, is it possible to handle events from pydeck like it is shown in the [Pydeck docs](https://deckgl.readthedocs.io/en/latest/event_handling.html) `Deck.deck_widget.on_click(my_callback)`?

**Comment by jourdain** [comment, marked as answer]
[This](https://github.com/Kitware/trame-deckgl/blob/master/vue-components/src/components/TrameDeck.js) will need to trigger some emit to enable such thing. But it should be possible.

**Reply by kerim371** [reply]
Thank you!
I will try to investigate this
